Dockerfile

```
@@ -93,4 +93,6 @@ RUN chmod -R go=u /app
 # Make all home files available (specifically .mix/)
 RUN chmod -R go=u $HOME
 
+HEALTHCHECK CMD sh -c "wget --no-verbose --tries=1 --spider http://localhost:\\$LIVEBOOK_PORT/public/health"
```

`LIVEBOOK_PORT` may not be set at runtime, so we need to use the default. I believe this we can do this:

```suggestion
HEALTHCHECK CMD wget --no-verbose --tries=1 --spider http://localhost:${LIVEBOOK_PORT-8080}/public/health || exit 1
```